modules/cognito/versions.tf

```
@@ -0,0 +1,12 @@
+terraform {
+  required_version = ">= 1.0.0"
+
+  required_providers {
+    aws = {
+      source = "hashicorp/aws"
+      # Starting with v4.8.0, the provider adds `aws_cognito_user_in_group` allowing adding Cognito Users to Cognito Groups in terraform
+      # https://github.com/hashicorp/terraform-provider-aws/releases
+      version = "~> 4.8.0"
```

```suggestion
      version = ">= 4.8.0"
```

This way we don't have to update the module when we want it to use `4.9`, `4.10`, or even `5.0`